app/src/main/java/com/aof/mcinabox/utils/FileTool.java

```diff
@@ -342,20 +339,7 @@ public static String readToString(String filePath) throws IOException {
      * [检查文件后缀名是否匹配]
      **/
     public static boolean FileSuffixFilter(String suffix, String fileName) {
-        int pos = -1;
-        for (int a = 0; a < fileName.length(); a++) {
-            if (fileName.charAt(a) == '.') {
-                pos = a;
-            }
-        }
-        if (pos == -1 || pos == 0) {
-            return false;
-        }
-        char[] theSuffixChars = new char[fileName.length() - pos - 1];
-        for (int a = pos + 1, b = 0; a < fileName.length(); a++, b++) {
-            theSuffixChars[b] = fileName.charAt(a);
-        }
-        return String.valueOf(theSuffixChars).equals(suffix);
+        return fileName.length() > 0 && fileName.charAt(0) != '.' && fileName.endsWith(suffix);
```

Comment:
This will not return false if "." is not present in the filename, however, the previous function did. Could you fix it? :)

Reply:
Thanks for reviewing the code, I fixed the mistake you mentioned in the latest commit (as well as a typecasting issue I introduced) ( ´△｀)

And the alternate background feature works properly now ﾍ(=￣∇￣)ﾉ
![Screenshot_20201227-124424](https://user-images.githubusercontent.com/24864674/103170053-97b18600-4841-11eb-824e-055211be467f.jpg)